actions/run.sh

```diff
@@ -1,21 +1,8 @@
 #!/bin/bash
 
-#: Title : run
-#: Date : 27/feb/2017
-#: Author : Daniel Ortega @gatero <me@daniel-ortega.mx>
-#: Version : 0.0.3
-#: Description : Run the project using docker-compose
-#: Options : 
-#:    -f) use your own docker-file
-#:    -d) default docker-compose
-#:    -h|*) help option
-
```

Comment:
Why the source documentation was removed ? it is modificable but not removable, you can add a collaborators tag and add your name or modify the functionality and usage documentation but not delete it